README.md

```diff
@@ -14,6 +14,7 @@ Quickstart
 ----------
 If you are using Maven with a BOM, add this to your pom.xml file.
 ```xml
+# [START with-bom]
```

Comment:
Snippet region tags need to be globally unique. The snippet tag should probably be something like `monitoring_install_with_bom`.

Also, if people will be copy/pasting this code, it should use xml comment format `<!-- comment here -->`

Reply:
Many other GCP products pull their client library instructions from this page for their instructions, for example, https://cloud.google.com/logging/docs/reference/libraries, https://cloud.google.com/error-reporting/docs/reference/libraries, https://cloud.google.com/bigquery/docs/reference/libraries. Thus, I though keeping the tag general would make it easier for those other products to pull in the instructions as well. 

Thoughts?

Reply:
This repository is only for the monitoring client. Each install instruction will be similar, but slightly different.

Reply:
Got it. Thanks! Will send a revised pr shortly.